cmd/buildah/containers.go

```diff
@@ -239,6 +239,10 @@ func outputContainers(store storage.Store, opts containerOptions, params *contai
 }
 
 func containerOutputUsingTemplate(format string, params containerOutputParams) error {
+	if !strings.HasPrefix(format, "{{.") || !strings.HasSuffix(format, "}}") {
+		return errors.Errorf("error format: %s", format)
```

Comment:
Can we change the error to ("error invalid format provided: %s", format)

Otherwise LGTM, thanks for the touchup!

Reply:
I'm afraid if someone will set `--format` to `"container ID: {{.Id}}"` this will return error although it's legit.

Reply:
I agree. if we want to do the format check to make sure their is json, it should be done with a regex expression.

Reply:
updated, PTAL.